sdk/test/Services/S3/IntegrationTests/CopyObjectPartTests.cs

```diff
@@ -0,0 +1,110 @@
+using Amazon.S3;
+using Amazon.S3.Model;
+using Amazon.S3.Util;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+
+namespace AWSSDK_DotNet.IntegrationTests.Tests.S3
+{
+    [TestClass]
+    public class CopyObjectPartTests : TestBase<AmazonS3Client>
+    {
+        private const string testContent = "This is the content body!";
+        private const string testKeyWithSlash = "/testsourcekey";
```

Comment:
There were no existing `CopyPart` integration tests, but it was being used in these two places:
- https://github.com/aws/aws-sdk-net/blob/8c6a77b03b0f967388d57e912bc67238f2c7ab19/sdk/test/Services/S3/IntegrationTests/KMSTests.cs#L536
- https://github.com/aws/aws-sdk-net/blob/8c6a77b03b0f967388d57e912bc67238f2c7ab19/sdk/test/Services/S3/IntegrationTests/ChecksumTests.cs#L328

I created this new file to verify the leading `/` works as expected.